builder/test/tests.sh

```diff
@@ -20,14 +20,16 @@ nmap --version
 lsof -v
 git --version
 vim --version
-pip --version
```

Comment:
Why not to keep pip 2?

Reply:
Depends on how much Pyhton2 we wish to keep. I could re-add pip for Python 2, but what would be the rationale? It's not like our users are going to be able to use Python 2 in any meaningful way.

Reply:
It's possible to write and run normal programs (even without ROS), right? It looks like a meaningful way. For older programs, for example.

Reply:
Okay, I've added installing `pip` for Python 2 back. `pip` will still default to `pip3`, users will still be able to use `pip2` for Python 2 packages.